shared-files/_glossary.md

```diff
@@ -0,0 +1,317 @@
+    </dd>
+    <p>
+        <b>Related terms:</b> <i>Downstream cluster, Imported cluster, Managed cluster, Registered cluster</i>
+    </p>
+</dl>
+
+## I
+
+<dl>
+    <dt>
+        Imported cluster
+    </dt>
+    <p>
+        <b>Versions:</b> Rancher v2.0–2.4
+    </p>
+    <dd>
+        An existing cluster that was provisioned by a third party, then set up to be managed by Rancher. <i>Registered clusters</i> in later versions of Rancher are roughly synonymous with imported clusters. The main difference is that there are more management features exposed in the Rancher UI for registered clusters than for imported clusters.
+    </dd>
+    <p>
+        <b>Related terms:</b> <i>Downstream cluster, Hosted cluster, Managed cluster, Reistered cluster</i>
+    </p>
+</dl>
+
+## K
+
+<dl>
+    <dt>
+        K3s
+    </dt>
+    <dd>
+        A lightweight Kubernetes distribution chiefly designed for workloads that run unattended under tight resource constraints.
+    </dd>
+    <p>
+        <b>Related terms:</b> <i>RKE, RKE2</i>
+    </p>
+</dl>
+
+## L
+
+<dl>
+    <dt>
+        Local cluster
+    </dt>
+    <dd>
+        An alternative name for a <i>Rancher server</i>, the Kubernetes cluster that hosts Rancher and manages other clusters. Here, <i>local</i> refers to the location of the Rancher installation, not to the user's local workstation. Local clusters can be hosted on-premises, in air-gapped environments, or in the cloud.
+    </dd>
+    <p>
+        <b>Synonyms:</b> <i>Rancher server, Upstream cluster</i>
+    </p>
+    <p>
+        <b>Related terms:</b> <i>Downstream cluster, Hosted cluster, Imported cluster, Managed cluster</i>
+    </p>
+</dl>
+
+## M
+
+<dl>
+    <dt>
+        Managed cluster
+    </dt>
+    <dd>
+        A Kubernetes cluster that is managed by a Rancher server. Managed clusters can run on a variety of infrastructure and can be located anywhere, including in the cloud, on-premises, or in air-gapped environments.
+    </dd>
+    <p>
+        <b>Synonyms:</b> <i>Downstream cluster</i>
+    </p>
+    <p>
+        <b>Related terms:</b> <i>Rancher server</i>
+    </p>
+</dl>
+
+## R
+
+<dl>
+    <dt>
+        Rancher 
+    </dt>
+    <dd>
+        A Kubernetes management tool to deploy and run clusters anywhere and on any provider. Rancher includes all the software and downstream cluster components used to manage the entire Rancher deployment.
+    </dd>
+    <dt>
+        Rancher Kubernetes Engine (RKE)
+    </dt>
+    <dd>
+        <ol>
+            <li> 
+                A CNCF-certified Kubernetes distribution that runs entirely within Docker containers. There are two main versions of RKE available. The 1.x software line is sometimes called <i>RKE1</i>. The next-generation line is called <i>RKE2</i>.
+            </li>
+            <li>
+                Shorthand for <i>RKE1</i>, the 1.x software line of RKE.
```

Comment:
Is this considered a second definition or is the ordered list used as a workaround to add another paragraph?

`Shorthand for RKE1` - it seems a bit awkward to call the preferred term a shorthand of a secondary form.
`the 1.x software line of RKE` - 

Also, both parts of the second listitem seemed like they can be dropped since the first listitem already makes a similar reference.

Reply:
It was intended as a second definition, but if we don't need it we can get rid of the list

Reply:
I'd vote for removing it. 

Forgot to include this in my original comment:
`the 1.x software line of RKE` - we're kind of defining RKE by making a reference to RKE with this part.

Reply:
If definition 2 stood alone without the previous definition describing RKE, it would be more confusing. But, at any rate, definition 1 already covers the RKE1 usage ("The 1.x software line is sometimes called <i>RKE1</i>") so the second definition isn't necessary. We can just remove the second definition and update that sentence in the first to, "The 1.x software line is sometimes called <i>RKE1</i>, or simply <i>RKE</i>."